Doc/library/urllib.request.rst

```diff
@@ -164,8 +164,8 @@ The :mod:`urllib.request` module defines the following functions:
    This helper function returns a dictionary of scheme to proxy server URL
    mappings. It scans the environment for variables named ``<scheme>_proxy``,
    in a case insensitive approach, for all operating systems first, and when it
-   cannot find it, looks for proxy information from Mac OSX System
-   Configuration for Mac OS X and Windows Systems Registry for Windows.
+   cannot find it, looks for proxy information from macOS System
```

Comment:
Is it the official name of the program?

Reply:
Don't think so

Reply:
On Apple's site it is called just "System Configuration".